data/pc/1.19.2/protocol.json

```diff
@@ -2132,7 +2139,7 @@
               ]
             },
             {
-              "name": "notTrustEdges",
+              "name": "suppressLightUpdates",
```

Comment:
why was this changed? was a field deleted and another one introduced in the same place ?